json4s/src/main/scala/io/finch/json4s/package.scala

```diff
@@ -0,0 +1,29 @@
+package io.finch
+
+import com.twitter.util.Try
+import io.finch.request.DecodeRequest
+import io.finch.response.EncodeResponse
+import org.json4s._
+import org.json4s.jackson.JsonMethods
+import org.json4s.jackson.Serialization._
+
+package object json4s {
+
+  /**
+   * @param formats json4s `Formats` to use for decoding
+   * @tparam A the type of data to decode into
+   */
+  implicit def decodeJson[A : Manifest](implicit formats: Formats): DecodeRequest[A] = DecodeRequest(
```

Comment:
Argh. Let's for with `Manifest` for now then. But please, add a TODO note that it has to be uodated.